### PR TITLE
hide user tags behind the toggle button

### LINKED ISSUE #5055: Hide user tags behind a toggle button
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/views/users/profile.html.erb](https://github.com/publiclab/plots2/blob/master/app/views/users/profile.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -121,7 +121,12 @@
   <div style="text-align:center;" class="visible-sm visible-xs">
     <img class="img-circle" id="profile-photo" style="width:50%;margin-bottom:20px;" src="<%= @user.photo_path %>" />
   </div>
-  <%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @user.id }", parent: :profile } %>
+
+  <p><a onClick="$('').toggle();" class="btn btn-default">User tags</a></p>
+
+  <div class="tags" style="display:none;">
+    <%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @user.id }", parent: :profile } %>
+  </div>
 
   <div class="profile_icons" style="text-align: center;">
     <% if @twitter.nil? == false %>
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by richa031**
Fixes #5055

I have made the required changes in the profile.html.erb file
please check if its done correctly this time.

### CODE REVIEW

**Comment by CleverFool77** [comment]
Agre with @divyabaid16 and @richa031 Can you put put a gif shoring the work. :)
Thanks

**Comment by CleverFool77** [comment]
I guess something similar happened in #5084. Now that you have actually made the PR. Can you reset it ?

**Comment by richa031** [comment]
I first deleted the branch I created then I created a new branch and only added the single file this time. I don't know what went wrong.

**Comment by gauravano** [comment]
Please link the issue you are trying to solve here. Thanks!

**Comment by divyabaid16** [comment]
@richa031 
First, you were at the master branch. Here you had modified your `Gemfile` and `Gemfile.lock`.
Then you created a new branch and had just added that file to commit. 
Now what happens is that the changes that are made locally is also being committed. So at master branch use `git stash`. By using this command `Gemfile` and `Gemfile.lock` won't be staged for commit and then switch the branch and only the required file will be added into the commit.

**Comment by richa031** [comment]
Thank you @divyabaid16 let me try this. @gauravano I'm trying to solve issue #5055

**Comment by gauravano** [comment]
Suggestion: Adding **Fixes #[issue-number]** in PR comment like this https://github.com/publiclab/plots2/pull/5144#issue-261917149 automatically closes the issue when PR gets merged so try to include that in your PR. Thanks!

**Comment by richa031** [comment]
Screenshot from 2019-03-19 23-29-21
This is what it is showing.

**Comment by gauravano** [comment]
@richa031 you can remove unwanted files after doing a commit by https://stackoverflow.com/questions/12481639/remove-files-from-git-commit 

or if you think the above one is complicated, then you can just copy the files from master - https://github.com/publiclab/plots2/blob/master/Gemfile and similarly for gemfile.lock. and then do the new commit. 


Thanks!

**Comment by richa031** [comment]
Screenshot from 2019-03-24 11-27-21
@gauravano  Done the required things as you said....but here is one more problem. Can you please help me with this

**Comment by gauravano** [comment]
You can force push your commits using `git push -f origin branch-2`

**Comment by SidharthBansal** [comment]
Please do as per Gaurav's suggestion

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -121,7 +121,13 @@
   <div style="text-align:center;" class="visible-sm visible-xs">
     <img class="img-circle" id="profile-photo" style="width:50%;margin-bottom:20px;" src="<%= @profile_user.photo_path %>" />
   </div>
-  <%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @profile_user.id }", parent: :profile, user: @profile_user } %>
+
+
+    <p><a onClick="$('').toggle();" class="btn btn-default">User tags</a></p>
+	<div class="tags" style="display:none;">
+		<%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @user.id }", parent: :profile } %>
```
Comment (gauravano) [inline app/views/users/profile.html.erb]:
```suggestion
		<%= render :partial => "tag/tagging", locals: { url: "/profile/tags/create/#{ @profile_user.id }", parent: :profile, user: @profile_user } %>
```

Comment (gauravano) [reply app/views/users/profile.html.erb]: Hey @richa031, you'll face this error when you try this PR. Also, after doing some changes, I think some more changes are required to complete. For now, do these changes, i will make some commits here. Thanks!

**Comment by gauravano** [comment]
Hi @richa031, you want any help? Thanks!

**Comment by SidharthBansal** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**